network-api/networkapi/wagtailpages/migrations/0004_delete_newspage.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.12 on 2022-02-25 01:54
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailinventory', '0002_pageblock_unique_constraint'),
+        ('wagtail_footnotes', '0004_footnote_translatable_mixin_migration'),
+        ('wagtailredirects', '0007_add_autocreate_fields'),
+        ('wagtailcore', '0066_collection_management_permissions'),
+        ('wagtailforms', '0004_add_verbose_name_plural'),
+        ('wagtailpages', '0003_auto_20220222_2022'),
```

Comment:
Django being overly zealous here

```suggestion
        ('wagtailpages', '0003_auto_20220222_2022'),
```